docs/use-cases/sensor-data.md

```diff
@@ -1,8 +1,56 @@
 ---
-title: "Capture and sync sensor or tabular data with an existing machine"
+title: "Capture and query sensor data"
 linkTitle: "Capture and query sensor data"
 weight: 30
 type: "docs"
-layout: "empty"
-canonical: "data/query/"
+description: "Query tabular data from your machine on the Viam app or from a compatible client."
 ---
+
+You can use the data management service to capture tabular data from a connected component on your machine and sync that data to the cloud.
+Once you have synced that data, you can query that data using {{< glossary_tooltip term_id="sql" text="SQL" >}} or {{< glossary_tooltip term_id="mql" text="MQL" >}} to search through that data for actionable insights.
+
+For example, you might configure data capture for several sensors on your machine, or across sensors on multiple machines, to report the ambient operating temperature.
+You could then run queries against that data to search for outliers or edge cases, to analyze how the ambient temperature affects your robots' operation.
+
+<table>
```

Comment:
At this point it may be worth turning this into shortcodes similar to how the cards and card shortcode work. Would you be interested in looking at that? You'll also want to look at how manualcard works.

```
{{< cards >}}
  {{% card link="LINK" %}}
  {{% card link="LINK" %}}
  {{% card link="LINK" %}}
{{< /cards >}
```

Reply:
Hmm, actually, I don't currently see the value proposition. A shortcode approach could more easily handle alternating image alignment? Or you want to convert these into squares instead of rectangles? Do you mean the `card` shortcode specifically, and not make-my-own new shortcode?

If the full-on card approach: It's hard to be limited to text that fits in 64x64 pixels, for examples. [These four cards](https://docs.viam.com/), for example, are quite constraining. I can't really fit some of these steps in a 1/6th the width (of current rectangular approach) box. I also can't link to more than one thing at a time per card, which really constrains each task -- for example, the single leading step currently would need to be three new cards: add DM, conf capture, conf sync.

I guess I'm asking: What is your vision for these use case pages? I can implement something to facilitate, sure -- but I think we look fine here -- what is missing that I should shortcode for?

Reply:
Hmm, you mean like [here](https://docs.viam.com/ml/vision/detection/#configure-an-mlmodel-detector)? I could see that working, but my age-old issue of fitting life lessons into 40 characters from 20 years ago resurfaces.

Reply:
OK, I have added a second option using cards exclusively. I feel that it loses a lot of good context and data.